packages/venia-concept/src/components/MiniCart/cartOptions.js

```diff
@@ -136,7 +125,14 @@ class CartOptions extends Component {
                     </span>
                 </div>
                 <div className={classes.form}>
-                    {options}
+                    <section className={classes.options}>
```

Comment:
Now that `isProductConfigurable` is used within `Options` to return `null` this looks cleaner. The only difference would be that we now always have a `<section>` element with `class="options"`. That doesn't matter because none of the css does any sizing with `section` or the class but it could so just a warning. Not a big deal :)

Reply:
Man after digging into how these are styled its not very clear. Looks like there is a `section` class that is used to compose the sections in this form _but_ the options class does not use it for some reason.

You don't have to do it here but I just wanted to note a better way forward would be to add `composes: section` to the options class, remove the margins, and add a similar section for composing within the miniCart.css class :D